src/sentry/api/serializers/models/alert_rule.py

```diff
@@ -92,6 +98,7 @@ def get_attrs(
     ) -> defaultdict[AlertRule, Any]:
```

Comment:
it might be nice to decompose this function a bit since it's almost 200 lines now.

one idea could be to split it up like: `get_attrs_triggers`, `get_attrs_activations` and split it up by each attribute being formatted.